src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let price = 40 * days;
+  const disc1 = 50;
+  const disc2 = 20;
```

Comment:
```suggestion
  const basicDiscount = 20;
```

give more details in variable name)